apps/src/lib/kits/maker/ui/EligibilityChecklist.jsx

```diff
@@ -86,6 +86,9 @@ export default class EligibilityChecklist extends Component {
       );
     }
 
+    // TODO: if we have an admin override, dont prompt for unit6 intention
+    // TODO: account for admin override in providing code
```

Comment:
Though this PR lets an admin override the application status, and update our model, the client isn't yet smart enough to handle this. I decided to put that off for a future PR, as this one was growing large.